doc/packaging-guide.xml

```diff
@@ -1,56 +1,77 @@
 <chapter xmlns="http://docbook.org/ns/docbook"
          xmlns:xlink="http://www.w3.org/1999/xlink"
-         xml:id="chap-quick-start">
+         xml:id="chap-packaging-guide">
 
-<title>Quick Start to Adding a Package</title>
+  <title>Packaging Guide</title>
 
-<para>To add a package to Nixpkgs:
+  <section xml:id="sec-policies">
 
-<orderedlist>
+    <title>Policies</title>
+
+    <para>The forum for policy discussion can be found at the <link
+xlink:href="https://github.com/NixOS/nixpkgs/labels/6.topic%3A%20policy%20discussion">
+issue tracker, topics are labeled with 'policy
+discussion'</link>. This section brewed from maintainer
+opinions. Regard the following paragraphs as drafts.</para>
+
+    <para>The nixpkgs package repository is not restrictive in
+comparison to other distributions. Non free packages are allowed but
+will not be built by Hydra. Packaging binaries is allowed, patching
```

Comment:
Thanks, I made a edit according to your suggestions. Could you also review that?